Editor/CoreAPI/CoreApi.cs

```diff
@@ -32,7 +32,7 @@ public class CoreApi
 
         public static CoreApi SharedInstance { get; } = new CoreApi();
 
-        internal CoreApi()
+        public CoreApi()
```

Comment:
As discussed in slack, these changes to `public` may be caused by us using `Editor.*` for our new namespaces instead of `AmazonGameLift.Editor`

Reply:
`CoreApi` is in the assembly `Editor.CoreApi`
`StateManager` is in the assembly `Editor.CoreApi`
However, StateManager constructor should be called from GameLiftPlugin in `Editor` assembly with `CoreApi` as a parameter, so needs to be public unless we restructure this